### PR TITLE
Add i18n for Klogg

### PR DESCRIPTION
**Pull request by nowhszh**
Newly added support for Chinese language, the translation part is not finished yet

### CODE REVIEW

**Comment by nowhszh** [comment]
I accidentally clicked the close button when merging conflicts， original pr #555

**Comment by nowhszh** [comment]
All development and translations for switching languages have been completed, currently klogg needs to be restarted after changing languages. the code that does not need to be restarted after switching languages I will create a new commit later

**Comment by nowhszh** [comment]
@variar  docker 22.04 looks like there's a problem with the build container

**Comment by nowhszh** [comment]
I tested the dockerfile for ubuntu 22.04 locally and it may be blocking here
image

**Comment by nowhszh** [comment]
> @variar docker 22.04 looks like there's a problem with the build container

fixed

File: src/app/i18n/zh-CN.ts [hunk 1]
```diff
@@ -4,278 +4,283 @@
 <context>
     <name>AbstractLogView</name>
     <message>
-        <location filename="../../ui/src/abstractlogview.cpp" line="424"/>
+        <location filename="../../ui/src/abstractlogview.cpp" line="425"/>
         <source>Copy the selection</source>
-        <translation type="unfinished">拷贝当前段</translation>
+        <translation>拷贝当前段</translation>
```
Comment (Junbo-Zheng) [inline src/app/i18n/zh-CN.ts:9]:
```suggestion
        <translation>复制所选内容</translation>
```

File: src/app/i18n/zh-CN.ts [hunk 2]
```diff
@@ -4,278 +4,283 @@
...
         <source>Marks and matches</source>
-        <translation type="unfinished"></translation>
+        <translation>标记和匹配</translation>
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="949"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="957"/>
         <source>Marks</source>
-        <translation type="unfinished"></translation>
+        <translation>仅标记</translation>
```
Comment (Junbo-Zheng) [inline src/app/i18n/zh-CN.ts:193]:
```suggestion
        <translation>标记</translation>
```

File: src/app/i18n/zh-CN.ts [hunk 3]
```diff
@@ -4,278 +4,283 @@
...
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="953"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="961"/>
         <source>Matches</source>
-        <translation type="unfinished"></translation>
+        <translation>仅匹配</translation>
```
Comment (Junbo-Zheng) [inline src/app/i18n/zh-CN.ts:198]:
```suggestion
        <translation>匹配</translation>
```

File: src/app/i18n/zh-CN.ts (hunk 2 above)
Comment (nowhszh) [reply src/app/i18n/zh-CN.ts:193]: The options in the context include "marks and match, marks, match",  so I think it's clearer to translate it as "marks only" here

File: src/app/i18n/zh-CN.ts (hunk 3 above)
Comment (nowhszh) [reply src/app/i18n/zh-CN.ts:198]: The options in the context include "marks and match, marks, match",  so I think it's clearer to translate it as "matchs only" here

File: src/app/i18n/zh-CN.ts (hunk 1 above)
Comment (nowhszh) [reply src/app/i18n/zh-CN.ts:9]: Your translation is better, I will update it later

**Comment by variar** [comment]
Thank you for all this work! I'll try the changes once and do the merge.

File: src/app/i18n/zh-CN.ts (hunk 1 above)
Comment (nowhszh) [reply src/app/i18n/zh-CN.ts]: For consistency with other translations, I have changed this to "拷贝选中内容"

**Review by Junbo-Zheng — APPROVED** [review]
@nowhszh Thanks for your explanation, LGTM. I can't wait to enjoy it.

**Comment by variar** [comment]
@nowhszh @Junbo-Zheng I've merged this and made a new build. Please check )

**Comment by Junbo-Zheng** [comment]
> @nowhszh @Junbo-Zheng I've merged this and made a new build. Please check )

Sorry for the late reply, Thanks @variar and @nowhszh.

**Pull request merged**